maat-court-data-api/src/test/java/gov/uk/courtdata/integration/contribution/ContributionsControllerIntegrationTest.java

```diff
@@ -146,15 +148,9 @@ void givenAInvalidContributionId_whenFindIsInvoked_thenCorrectErrorResponseIsRet
     void givenAValidParameter_whenFindIsInvoked_theCorrectResponseIsReturned() throws Exception {
         MvcResult result = runSuccessScenario(MockMvcRequestBuilders.get(ENDPOINT_URL + "/" + TestModelDataBuilder.REP_ID)
                 .contentType(MediaType.APPLICATION_JSON));
-
-        ContributionsEntity currentEntity = objectMapper.readValue(result.getResponse().getContentAsString(), ContributionsEntity.class);
-
-        contributionsEntity.setContributionCap(currentEntity.getContributionCap());
-        contributionsEntity.setMonthlyContributions(currentEntity.getMonthlyContributions());
-        contributionsEntity.setUpfrontContributions(currentEntity.getUpfrontContributions());
-
+        List<ContributionsEntity> contributionsEntityList = contributionsRepository.findAllByRepId(TestModelDataBuilder.REP_ID);
```

Comment:
Can you add one more test for findLatestContribution

Reply:
Thanks @skasthuri36326 : Added now